modules/crd-creating-crds.adoc

```diff
@@ -28,14 +28,29 @@ metadata:
 spec:
   group: stable.example.com <3>
   versions:
-    name: v1 <4>
+    - name: v1 <4>
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                cronSpec:
+                  type: string
+                image:
+                  type: string
+                replicas:
+                  type: integer
   scope: Namespaced <5>
   names:
     plural: crontabs <6>
     singular: crontab <7>
     kind: CronTab <8>
     shortNames:
-    - ct <9>
+     - ct <9>
```

Comment:
still have one whitespace, please delete it.

Reply:
Fixed https://github.com/openshift/openshift-docs/pull/71969/files#diff-7ddc0a682fe2c5e55c86cd502c63bb313c7509f8dbb35b318a5ffd3a96a9ad8fL37-L38